src/app/tool-editor/object-inspector/common-sections/description-section/description.component.ts

```diff
@@ -165,4 +171,30 @@ export class DescriptionComponent extends DirectiveBase implements ControlValueA
     isFileType() {
         return this.port.type.type === "File" || this.port.type.items === "File";
     }
+
+    setDisabledState(disabled: boolean) {
```

Comment:
How about:
Object.keys(this.form.controls).forEach((item) => {
const control = this.form.controls[item];
disabled ? control.disable() : control.enable();
})